assets/locales/cs.json

```diff
@@ -376,35 +377,35 @@
       "ADDLAYERS": "Vyberte data nebo služby pro vaši mapovou kompozici",
       "APPLICATIONSETTINGS": "Zadejte uživatelské nastavení aplikace",
       "DRAW": "Nakreslete nové prvky",
-      "FEATURE_TABLE": "Zobrazite prvky vrstvy v html tabulce",
+      "FEATURE_TABLE": "Zobrazte prvky vrstvy v tabulce",
```

Comment:
I would rather use "Zobrazit" instead of "Zobrazte"

Reply:
Replys in Czech will be more efficient here I suppose :-)
Použil jsem "Zobrazte" protože i popisy ostatních panelů začínají stejným časem a osobou: "Zadejte ...", "Vyberte ..." Takže mi jde o jednotnost. Buď bych dal všechno ve 2. os. č. mn. nebo všechno v infinitivu.

Reply:
Rozumím, ale zase jde o kontext. Jsou tady také pojmy jako "Zobrazit", "Seřadit", apod. Přijde mi logické, že u akcí, kdy na něco mám kliknout a něco se stane, se použije infinitiv. Pokud mám zadávat nějaký vstup, pak ve druhé os. Co myslíš?

Reply:
Tyhle řetězce jsou popisky toho, co který panel umí. Není to přímo akce. Zobrazují se při najetí myší na ikonu v panelu.
Nejvíc by se tu asi hodila 3. os.? Jako "Zobrazuje prvky vrstvy v tabulce" nebo "Kreslí nové prvky"?

Reply:
Z pohledu češtiny asi ano, ale myslím, že by to působilo divně :-) Koukám, že i v EN textech je tam infinitiv, tak bych ho tam nechal i v CS. Ona to de facto akce je, např. Zobrazit seznam kompozic...klik a zobrazí se ti panel kompozic.

Reply:
No... to je problém, že v EN jsou infinitiv, první os. i druhá os. úplně stejné. Vlastně i ta 1. by tady dávala smysl, jakože ten panel popisuje sám sebe: "Zobrazuji prvky vrstvy v tabulce" atp. 😄 Ale je to jen na nás, jak nám to bude v CS znít přirozeně. Jestli infinitiv, tak opravím ty popisky všechny do infinitivu.

Reply:
Přizval jsem manželku jako laického uživatele a prý infinitiv :D

Reply:
UX testing, wow 😄 Ok, tak infinitiv.